CondFormats/DataRecord/src/ES_Darkening.cc

```diff
@@ -0,0 +1,4 @@
+#include "FWCore/Utilities/interface/typelookup.h"
+#include "CondFormats/HcalObjects/interface/HBHEDarkening.h"
+
+TYPELOOKUP_DATA_REG(HBHEDarkening);
```

Comment:
ES_Darkening.cc seems like a bad way to call this file in a generic area (where other ES files are for a different detector)